_includes/head-idr.html

```diff
@@ -45,7 +45,7 @@
                             <li><a href="{{ "/api.html" | prepend: site.baseurl }}">API Access</a></li>
                             <li><a href="{{ "/download.html" | prepend: site.baseurl }}">Data download</a></li>
                             <li><a href="{{ "/itr.html" | prepend: site.baseurl }}">Image Tools Resource (ITR)</a></li>
-                            <li><a href="/jupyter">Virtual Analysis Environment (VAE)</a></li>
+                            <li><a href="{{ "/linked-environments.html" | prepend: site.baseurl }}">Analysis Environments</a></li>
```

Comment:
At the moment this PR uses a mixture of `Environments`, `Analysis Environments`, `linked-environments` and we should unify. Proposing `Analysis Environments/analysis-environments` unless someone has a better suggestion.